packages/migrate/src/__tests__/fixtures/schema-only-cockroachdb/prisma/shadowdb.prisma

```diff
@@ -1,11 +1,7 @@
-generator client {
-  provider = "prisma-client-js"
-}
-
 datasource db {
   provider          = "cockroachdb"
-  url               = env("TEST_COCKROACHDB_SHADOWDB_URI_MIGRATE")
-  shadowDatabaseUrl = env("TEST_COCKROACHDB_SHADOWDB_URI_MIGRATE")
+  url               = env("TEST_COCKROACH_SHADOWDB_URI_MIGRATE")
+  shadowDatabaseUrl = env("TEST_COCKROACH_SHADOWDB_URI_MIGRATE")
```

Comment:
I see no equivalent renaming of an env var definition anywhere - did I miss it?

Reply:
So how did this test ever work? Should it not have failed for a missing env var before? Something doesn't add up, right?

Reply:
The answer is pretty easy: `shadowdb.prisma` was never loaded before. In other words, this pre-existing schema was just noise in the test codebase

Reply:
Ok, that explains it 👍 Good we fixed that then.